File: mvn-advent-of-code/src/test/java/advent/twenty_fifteen/Day21RpgSimulator20xxTest.java
package advent.twenty_fifteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day21RpgSimulator20xxTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day21RpgSimulator20xx("/twenty_fifteen/Day21-RpgSimulator20xx-input.txt");

        assertEquals(91, dp.getPart1Answer().intValue());
        assertEquals(158, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_fifteen/Day21RpgSimulator20xxTest.scala
package advent.twenty_fifteen

import org.scalatest.{FlatSpec, Matchers}

class Day21RpgSimulator20xxTest  extends FlatSpec with Matchers {
  "2015 Day 21 - Input" should "calculate answers" in {
    val dp = new Day21RpgSimulator20xx("/twenty_fifteen/Day21-RpgSimulator20xx-input.txt")
    dp.part1Answer should be(91)
    dp.part2Answer should be(158)
  }
}
